tests/telemetry/test_telemetry.py

```diff
@@ -12,6 +12,31 @@
 from conftest import _write_sample_conda_env, _prepare_files
 
 
+# The below fixtures are to mock the different virtual environments
+# Ref: https://stackoverflow.com/questions/51266880/detect-if-
+# python-is-running-in-a-conda-environment
```

Comment:
thanks for including the reference, i think this and the next one should go next to the function that checks the conditions, rather than the fixtures